app/services/clients/publishing_api.rb

```diff
@@ -26,8 +26,18 @@ def fetch_all(fields)
         .flatten
     end
 
-    def fetch(content_id, locale)
-      normalise(publishing_api.get_content(content_id, locale: locale))
+    def fetch(content_id, options = {})
+      normalise(publishing_api.get_content(content_id, options))
+    end
+
+    def fetch_latest_published(content_id, locale)
+      content_item = fetch(content_id, locale: locale)
+      return content_item if content_item[:publication_state] == "published"
+
+      preceding_version = content_item[:user_facing_version] - 1
```

Comment:
Why is this condition being checked? If the version is 0, shouldn't it be a `non-published` content item. Have you found this scenario in our current data?

Reply:
`user_facing_version` [is 1-based](https://github.com/alphagov/publishing-api/blob/ad08eeb43b4e92d7823017812ff7d3007bddff42/db/migrate/20161214113229_pillar_column_defaults.rb#L5). If the `preceding_version` is 0, then the content item we have is actually the first version (and therefore there is no preceding version).